docs/index.md

```diff
@@ -88,7 +88,7 @@ ABM features users have shared that you may want to use in your model
 
 Mesa Overview <overview>
 tutorials/intro_tutorial
-tutorials/visualization_tutorial
+Examples <examples>
```

Comment:
Don't we still need the visualization on the index as well? (I am adding an seminal examples link to the intro

Reply:
This is a left over from the problems building the docs with the visualization tutorial errorring out. So yes, you are right.